lib/fennec/udp/worker_supervisor.ex

```diff
@@ -4,14 +4,14 @@ defmodule Fennec.UDP.WorkerSupervisor do
 
   alias Fennec.UDP
 
-  def start_link(base_name) do
+  def start_link(opts, base_name) do
```

Comment:
Little nitpick, Elixir has this convention of putting least important arguments (and options, I believe) as a last argument to a function.